apps/neuvector/neuvector/prod/01.yaml

```diff
@@ -30,7 +30,15 @@ spec:
           secretName: nv-storage-secret
           shareName: neuvector-data-01
         ingress:
-          host: neuvector01-api.service.core-compute-prod.internal
+          host: cft-neuvector01-api.platform.hmcts.net
+          enabled: true
+          path: "/"
+          annotations:
+            kubernetes.io/ingress.class: traefik
+            traefik.ingress.kubernetes.io/router.tls: "true"
+        apisvc:
+          annotations:
+            traefik.ingress.kubernetes.io/service.serversscheme: https
```

Comment:
I think all these properties can be moved to https://github.com/hmcts/cnp-flux-config/pull/22280/files#diff-a2009d99fd90ff31dd13559f6d278e12c338516e12c16230c7871d3cce61c3cdR88 to remove duplication.

Reply:
I've opened another PR to use Kustomise to remove dupe spec - https://github.com/hmcts/cnp-flux-config/pull/22281